pkg/status/render_test.go

```diff
@@ -7,11 +7,13 @@ package status
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestFormatStatus(t *testing.T) {
+	time.Local = time.UTC
```

Comment:
This is for avoiding the failure on [Windows unit tests](https://github.com/DataDog/datadog-agent/actions/runs/3754008433/jobs/6377831240).